docs/pbc/all/search/202212.0/third-party-integrations/configure-algolia.md

```diff
@@ -106,6 +156,34 @@ Default attributes for faceting are as follows:
 
 The `prices` attribute is an object with nested fields. Algolia creates facets for each nested field and creates facets for all the currencies and pricing modes available in product entities.
 
+#### Facet configuration
+
+##### Searchable
+
+Attributes defined as searchable may be used while calling Algolia's `searchForFacetValues` method, which can be used for Yves integration. This method is necessary to display catalog page facets if many values are possible for each facet - in this case, only 100 of those values will be shown by default. Accessing other values requires searching for them using the method above. Select this option if you plan on having a large number of different values for this facet, and use Yves.
```

Comment:
"Accessing other values requires searching for them using the method above."  - what is the "method above" exactly?

Reply:
"method above" is `searchForFacetValues` mentioned earlier in the text

"and use Yves" - in this case we mean that this is applicable specifically if customer uses Yves. But overall this is applicable to any frontend.

Reply:
We need to rename Yves into Storefront